client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -84,37 +71,29 @@ const professionalEmailCardInformation: ProviderCard = {
 };
 
 const ProfessionalEmailCard: FunctionComponent< EmailProvidersStackedCardProps > = ( props ) => {
-	const { currencyCode = '', hasCartDomain, domain, selectedDomainName, titanMailProduct } = props;
-	const professionalEmail = professionalEmailCardInformation;
+	const {
+		currencyCode = '',
+		hasCartDomain,
+		detailsExpanded,
+		domain,
+		onExpandedChange,
+		selectedDomainName,
+		intervalLength,
+		titanMailMonthlyProduct,
+		titanMailAnnuallyProduct,
+		comparisonContext,
+		productsList,
+		shoppingCartManager,
+		selectedSite,
+		source,
+	} = props;
+	const professionalEmail: ProviderCard = professionalEmailCardInformation;
+	professionalEmail.detailsExpanded = detailsExpanded;
```

Comment:
As above, this line is changing the global `professionalEmailCardInformation`, which probably (?) wasn't intentional.

You can make a copy of that object to mutate instead by changing the previous line to:

```
	const professionalEmail: ProviderCard = {...professionalEmailCardInformation};
```

Reply:
Good catch, thank you Payton!